css/css-borders/tentative/corner-shape/corner-shape-angle-001.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>CSS Borders and Box Decorations 4 Test: 'corner-shape: angle'</title>
+<link rel="author" title="Sebastian Zartner" href="mailto:sebastianzartner@gmail.com">
+<link rel="help" href="https://drafts.csswg.org/css-borders-4/#corner-shaping">
+<link rel="match" href="../../reference/corner-shape-angle-ref.html">
```

Comment:
I'm extremely skeptical that you're going to get an exact pixel match between CSS borders and SVG, in the presence of antialiasing.

Maybe the only real answer is that this test will eventually be `fuzzy` once the feature is implemented, as I mentioned above.

(The deeper risk is that a browser developer implements the feature, finds the tests that test the feature by seeing what WPT tests pass once the feature is implemented, and misses these tests because they *don't* start passing... and then the feature later regresses in ways that these tests would have caught if they had been marked as passing when the feature was initially implemented.)

Reply:
> (The deeper risk is that a browser developer implements the feature, finds the tests that test the feature by seeing what WPT tests pass once the feature is implemented, and misses these tests because they don't start passing... and then the feature later regresses in ways that these tests would have caught if they had been marked as passing when the feature was initially implemented.)

I assumed that implementors explicitly search for the tests that check the feature they are implementing. And with the name "corner-shape" in the file names it should be clear enough. Though I see your point here.

As noted above, I could add fuzzy matching, though I'd need help to find proper values for `maxDifference` and `totalPixels`.

Reply:
> though I'd need help to find proper values for maxDifference and totalPixels.

The maxDifference and totalPixels values are and would be indicated in the bottom-left corner of the Compare [screenshots] webpage. An example with an entirely randomly chosen test with failures :

Go to
https://wpt.fyi/results/css/css-backgrounds/background-position-negative-percentage-comparison-002.html
Click one of the Compare links
Look at bottom left corner for those values.